vdirsyncer/__init__.py

```diff
@@ -2,7 +2,12 @@
 '''
 vdirsyncer is a synchronization tool for vdir. See the README for more details.
 '''
-__version__ = '0.5.1'
+
+try:
+    import pkg_resources
+    __version__ = pkg_resources.require('vdirsyncer')[0].version
```

Comment:
use get_distribution, by the point that that code runs, the distribution is already activated for certain